### ISSUE TITLE
JSON to TS  a better alternative

### ISSUE THREAD

**Issue by MariusAlch** [johnpapa/vscode-angular-essentials#3]
Plain and simple, was not satisfied with what Json2ts had to offer, so I created my own json parser (spent a lot of time) to do a better job (to my knowledge there is no unhandled edge cases), than Json2ts which is full of erroneous edge cases.

comparison: https://ibb.co/fwnoYF

So removing it from the recommendation list after hearing one side of the story + being blamed for copying doesnt feel right.
Either way, I just want you to make a rational decision which one you want to use for this list.

references: 
JSON to TS: https://marketplace.visualstudio.com/items?itemName=MariusAlchimavicius.json-to-ts
Json2ts: https://marketplace.visualstudio.com/items?itemName=GregorBiswanger.json2ts

**Comment by johnpapa** [comment]
The contents of the pack may change more. I wrote in my blog post that it's an early alpha release. Some don't work well together. Some have bugs. Others are not used as often. And some I just don't use that much. I reserve the right to add or remove as per what I use. 

It's good that there are similar extensions. It gives people choices. I chose to try the other one. 

Thanks for making the one you created.
